test/rules/std_L003_L036_L039_combo_test.py

```diff
@@ -4,12 +4,14 @@
 and L039 (unnecessary white space) can result in incorrect indentation.
 """
 
+import textwrap
+
```

Comment:
Do we need this new line:

```suggestion
```

Reply:
it's pretty standard to separate import groups into standard library, 3rd party , and local packages, each separated by a new line so I'd leave this one in here